core/build.gradle

```diff
@@ -31,19 +31,22 @@ mrJarVersions.each { version->
         options.release.set(targetJavaVersion)
     }
 
-    task "testJava${version}"(type: Test, group: 'Verification', description: "Runs unit tests for Java ${version} source set") {
+    tasks.register("testJava${version}", Test) {
+        group = 'Verification'
+        description = "Runs unit tests for Java ${version} source set"
         testClassesDirs = sourceSets."java${version}Test".output.classesDirs
         classpath = sourceSets."java${version}Test".runtimeClasspath
 
+        project.ext.configureFlakyTests(it)
```

Comment:
optional; I haven't tried this myself, but I'm wondering if there's a reason not to merge `configureFlakyTests` into `configureCommonTestSettings` now that all test tasks are using the junit api regardless of the engine

Reply:
I think it's a good idea. Let me try that. 👍

Reply:
Addressed. 😉 
Will send a PR to line/gradle-scripts as well.